packages/peregrine/lib/talons/CheckoutPage/ShippingInformation/AddressForm/useGuestForm.js

```diff
@@ -45,7 +45,8 @@ export const useGuestForm = props => {
                         address: {
                             ...address,
                             // region_id is used for field select and region is used for field input
```

Comment:
I wonder if this comment is still relevant, as it looks like we are removing `region.region`. Perhaps update comment to reflect what is now happening